src/logic/Windows.swift

```diff
@@ -114,6 +114,14 @@ class Windows {
         return list.count > focusedWindowIndex ? list[focusedWindowIndex] : nil
     }
 
+    static func minimizeOtherWindows() {
+        Windows.list.enumerated().forEach {
+            if $0.offset != focusedWindowIndex && !$0.element.isMinimized && !$0.element.isWindowlessApp && $0.element.shouldShowTheUser {
+                $0.element.minDemin()
```

Comment:
You check `shouldShowTheUser` here. Is it what the user would expect though? Let's imagine I use the ``` command+` ``` shortcut. It shows me only windows of the active app. When I set it to "minimize other windows" on release, I think I expect that every other window on my screen will minimize, except the one I just focused, no? Here it will only minimize the other windows of the active app.

Same logic will keep some windows on-screen, depending on the shortcut filtereing logic behind `shouldShowTheUser`.

Another point: I wonder if people will expect this single-window mode to be per-Space and/or per-Screen. I'm not sure users expect windows in every Space and on every Screen to minimize when they go for this type of workflow. Maybe they would like to have a "main window" per Screen?

What do you think?